CMakeLists.txt

```diff
@@ -25,6 +25,10 @@ cmake_policy(SET CMP0135 NEW) # policy to set the timestamps of extracted conten
 project(nifi-minifi-cpp VERSION 0.99.0)
 set(PROJECT_NAME "nifi-minifi-cpp")
 
+if (NOT CMAKE_BUILD_TYPE)
+    set(CMAKE_BUILD_TYPE "Release" CACHE STRING "Built type" FORCE)
```

Comment:
```suggestion
    set(CMAKE_BUILD_TYPE "Release" CACHE STRING "Build type" FORCE)
```

Reply:
Fixed in d820cad82ffc9c44fbe2a34d1ad6ad6d8a9b013d